cpp/ql/src/Diagnostics/SuccessfulExtractions.ql

```diff
@@ -0,0 +1,15 @@
+/**
+ * @name Successfully extracted files
+ * @description Lists all files in the source code directory that were extracted without encountering an error.
+ * @kind diagnostic
+ * @id cpp/diagnostics/successfully-extracted-files
```

Comment:
This file and `FailedInvocations.ql` have unnecessarily different filenames and id/name.

Reply:
No, I don't think so: `SuccessfulExtractions` and `FailedExtractions` divide the set of files in the database (excluding files outside the source root) into two disjoint sets, with their union being all files in the database.
This is what (I think) you want as a user - a file is either successfully extracted or not.

`FailedInvocations.ql` is another beast - it is useful for (internal, and maybe also external) debugging, but it doesn't try to report things on a file level.
This is why I want a different filename to make it totally clear that the query has not much to do with the other two.

I'll give the names and id's of the queries another shot though, I'm not happy with them, now that I look at them again.

Reply:
What I mean is that I'd expect the id of the query to mirror its file name. So when the query has id `cpp/diagnostics/successfully-extracted-files`, why isn't its file named `Diagnostics/SuccessfullyExtractedFiles.ql`?